diff --git a/moshi/pyproject.toml b/moshi/pyproject.toml
index 4136078..f185d41 100644
--- a/moshi/pyproject.toml
+++ b/moshi/pyproject.toml
@@ -6,7 +6,7 @@ dependencies = [
     "numpy >= 1.26, < 2.3",
     "safetensors >= 0.4.0, < 0.7",
     "huggingface-hub >= 0.24, < 0.34",
-    "bitsandbytes >= 0.45, < 0.46; sys_platform == 'linux'",
+    "bitsandbytes >= 0.45; sys_platform == 'linux'",
     "einops >= 0.7, < 0.9",
     "sentencepiece == 0.2",
     "sounddevice == 0.5",
